app/jobs/pollable_job_wrapper.rb

```diff
@@ -36,7 +37,11 @@ def after(job)
       def convert_to_v3_api_error(exception)
         v3_hasher = V3ErrorHasher.new(exception)
         error_presenter = ErrorPresenter.new(exception, Rails.env.test?, v3_hasher)
-        YAML.dump(error_presenter.to_hash)
+        res = YAML.dump(error_presenter.to_hash)
+        return res if res.size <= 16_000 || exception.backtrace.empty?
```

Comment:
Think `bytesize` would be a little safer in case of unicode characters

Reply:
or is the db column support unicode and arbitrarily limit on number of characters?

Reply:
Yeah, good point. I'll get that in,after cli standup